openapi/integration/aws/users/get-user.json

```diff
@@ -12,7 +12,7 @@
         "method.response.header.Access-Control-Allow-Origin": "'*'"
       },
       "responseTemplates": {
-        "application/json": "#set($inputRoot = $input.path('$'))\n#if($inputRoot.Items.size() == 0)\n{\n \"success\": false,\n \"message\": \"No user data found\"\n}\n#set($context.responseOverride.status = 404)\n#else\n{\n \"success\": true,\n \"data\": {\n #foreach($item in $inputRoot.Items)\n #if($item.SK.S == 'PROFILE')\n \"phoneNumbers\": [\n #foreach($phone in $item.phoneNumbers.L)\n \"$phone.S\"#if($foreach.hasNext),#end\n #end\n ],\n \"name\": \"$item.name.S\",\n \"bloodGroup\": \"$item.bloodGroup.S\",\n \"lastDonationDate\": \"$item.lastDonationDate.S\",\n \"height\": $item.height.S,\n \"weight\": $item.weight.S,\n \"gender\": \"$item.gender.S\",\n \"dateOfBirth\": \"$item.dateOfBirth.S\",\n \"availableForDonation\": \"$item.availableForDonation.S\",\n \"NIDFront\": \"$item.NIDFront.S\",\n \"NIDBack\": \"$item.NIDBack.S\",\n \"lastVaccinatedDate\": \"$item.lastVaccinatedDate.S\",\n #end\n #end\n \"preferredDonationLocations\": [\n #foreach($location in $inputRoot.Items)\n #if($location.SK.S.startsWith('LOCATION#'))\n {\n \"area\": \"$location.area.S\",\n \"city\": \"Dhaka\",\n \"latitude\": $location.latitude.N,\n \"longitude\": $location.longitude.N\n }#if($foreach.hasNext),#end\n #end\n #end\n ]\n },\n \"message\": \"User data retrieved successfully\"\n}\n#end"
+        "application/json": "#set($inputRoot = $input.path('$'))\n#if($inputRoot.Items.size() == 0)\n{\n \"success\": false,\n \"message\": \"No user data found\"\n}\n#set($context.responseOverride.status = 404)\n#else\n{\n \"success\": true,\n \"data\": {\n #foreach($item in $inputRoot.Items)\n #if($item.SK.S == 'PROFILE')\n \"phoneNumbers\": [\n #foreach($phone in $item.phoneNumbers.L)\n \"$phone.S\"#if($foreach.hasNext),#end\n #end\n ],\n \"name\": \"$item.name.S\",\n \"bloodGroup\": \"$item.bloodGroup.S\",\n \"lastDonationDate\": \"$item.lastDonationDate.S\",\n \"height\": $item.height.S,\n \"weight\": $item.weight.S,\n \"gender\": \"$item.gender.S\",\n \"dateOfBirth\": \"$item.dateOfBirth.S\",\n \"availableForDonation\": \"$item.availableForDonation.S\",\n \"NIDFront\": \"$item.NIDFront.S\",\n \"NIDBack\": \"$item.NIDBack.S\",\n \"lastVaccinatedDate\": \"$item.lastVaccinatedDate.S\",\n #end\n #end\n \"preferredDonationLocations\": [\n #set($isFirstIter = true)\n #foreach($location in $inputRoot.Items)\n #if($location.SK.S.startsWith('LOCATION#'))\n #if(!$isFirstIter),#end\n #set($isFirstIter = false)\n {\n \"area\": \"$location.area.S\",\n \"city\": \"Dhaka\",\n \"latitude\": $location.latitude.N,\n \"longitude\": $location.longitude.N\n }\n #end\n #end\n ]\n },\n \"message\": \"User data retrieved successfully\"\n}\n#end"
```

Comment:
should this not include our cfid, requestid etc for logging

Reply:
yes, we need to log these in apigateway. cause we are not using any lambda here. but haven't worked on it yet. haven't got the time. I'm gonna create a ticket for that

Reply:
owwh, no lambdas, right nevermind